java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -1389,7 +1444,15 @@ class DatabaseImpl(
         require(primitiveValue is ReferencablePrimitive<*>) {
             "Expected value to be a ReferencablePrimitive but was $primitiveValue."
         }
-        val value = primitiveValue.value
+        return getPrimitiveValueIdForRaw(primitiveValue.value, typeId, db, counters)
+    }
+
+    private fun getPrimitiveValueIdForRaw(
```

Comment:
why this refractor if you are not using this function anywhere else?

Reply:
Ah, this is a remnant. I'll remove it.